demoapp/package.json

```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "cordova-android": "^7.0.0",
-    "cordova-ios": "^4.5.4",
+    "cordova-ios": "^5.0.0",
     "cordova-plugin-appcenter-analytics": "^0.3.3",
     "cordova-plugin-appcenter-crashes": "^0.3.3",
```

Comment:
We should consider using exact versions to avoid potential issues we had in React Native: https://github.com/Microsoft/AppCenter-SDK-React-Native/issues/449.